webapp/src/js/services/android-api.js

```diff
@@ -138,8 +139,21 @@ angular.module('inboxServices').factory('AndroidApi',
           // If we're viewing a tab, but not the primary tab, go to primary tab
           var primaryTab = $('.header .tabs').find('> a:visible:first');
           if (!primaryTab.is('.selected')) {
-            $state.go(primaryTab.attr('ui-sref'));
-            return true;
+            var uiSref = primaryTab.attr('ui-sref');
+            if (uiSref) {
+              $state.go(primaryTab.attr('ui-sref'));
+              return true;
+            } else {
+              var message = 'Attempt to back to an undefined state';  
```

Comment:
To help with debugging next time include some more information like that your in the android api back function.